### PR TITLE
Azure updates 2017-02-07

### PR DESCRIPTION
**Pull request by russellseymour**
Multiple updates to fix minor issues and provide more feedback when deploying to Azure

- Infranodes can now be built correctly as have sorted the issue with `ssh_pty`
- In case it is relevant the `sysprep` command for all Windows machines has been separated from the `cd` command
- Corrected issue in Wombat that prevented the use of a different wombat configuration file
- Added tags to the Azure resource group based on the `owner` if it has been specified in the `wombat.yml` file.  Will also add `InUse: true` so that it is not deleted by clean up routines
- Added `--config` argument to the `update` command as it was missing and therefore could not specify a different `wombat.yml` file
- Corrected syntax error in ARM template that prevented deployment
- By default a deployment to Azure will run synchronously and give output in the terminal.  However this can be overridden using `--async` on the command line.  (Thanks to @stuartpreston form much of this code)
- Corrected issue when generating template that meant the URI to the custom image for the infranode was not being set correctly.

### CODE REVIEW

File: lib/wombat/build.rb [hunk 1]
```diff
@@ -95,6 +95,19 @@ def prepare_azure()
...
         resource_group.location = wombat['azure']['location']
 
+        # Create hash to be used as tags on the resource group
+        tags = {
+          InUse: "true"
+        }
+
+        # If an owner has been sepcified in the wombat file then add this as a tag
+        if wombat.key?('owner') && !wombat['owner'].nil?
+          tags[:owner] = wombat['owner']
```
Comment (binamov) [inline lib/wombat/build.rb:105]: may I suggest grabbing the Env Var for the operator's username here automagically, unless overridden by the `owner` tag in wombat.yml?

**Comment by binamov** [comment]
looks fine

File: lib/wombat/build.rb (hunk 1 above)
Comment (binamov) [reply lib/wombat/build.rb:105]: one should expect people will share their wombat.yml and no one will change the owner tag in the process

Comment (russellseymour) [reply lib/wombat/build.rb:105]: Good idea.  Will make the necessary changes.

**Pull request merged**